frontend/src/store/playerElement.ts

```diff
@@ -115,6 +122,17 @@ class PlayerElementStore {
     }
   };
 
+  private _isSupportedFonts = (mimeType: string): boolean => {
+    // ttf,otf,woff,woff2
```

Comment:
```suggestion
  private _isSupportedFont = (mimeType: string): boolean => {
```

We're checking a single font here
The comment is useless given how concise the function is, imo.